pyproject.toml

```diff
@@ -62,11 +62,11 @@ maintainers = [
 # Note: Python and NumPy upper version bounds should be set correctly in
 # release branches, see:
 #     https://scipy.github.io/devdocs/dev/core-dev/index.html#version-ranges-for-numpy-and-other-dependencies
-requires-python = ">=3.9"
+requires-python = ">=3.9,<3.12"
```

Comment:
We're going to support Python 3.12 on this maintenance branch, and I suspect it already works. So let's put it at 3.13 already?